kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/configuration/Password.java

```diff
@@ -13,24 +13,26 @@
 
 public class Password 
 {
-	private char[] password;
+	protected char[] m_password;
```

Comment:
@ibinshtok why we need to widen the visibility? I've seen that in WifiPassword.java you are using m_password but, wouldn't it better to keep the password variable private and use the public getPassword to return the password in the subclass?